lib/mongodb.js

```diff
@@ -1321,7 +1322,7 @@ MongoDB.prototype.count = function count(model, where, options, callback) {
     debug('count', model, where);
   }
   where = self.buildWhere(model, where);
-  this.execute(model, 'count', where, function(err, count) {
+  this.execute(model, 'countDocuments', where, function(err, count) {
```

Comment:
Checked the `countDocuments` function in the [mongodb driver module](https://github.com/mongodb/node-mongodb-native), seems it returns a `Promise` instead of taking a callback.
I'd also like to hear more details from @gendigbadig about why changing it here.

Reply:
As I discovered, the `count` method has been deprecated and the recommendation is to switch to `countDocuments`. As such, I'm ok with this change in this PR.